Documentation/app-operations.md

```diff
@@ -0,0 +1,197 @@
+   Loaded: loaded (/usr/lib64/systemd/system/mysql.service; static; vendor preset: enabled)
+      Active: active (running) since Wed 2016-07-09 17:00:50 UTC; 8min 38s ago
+       Main PID: 6 (mysqld)
+          CGroup: /machine.slice/machine-rkt\x2d11efeefb\x2d2e8d\x2d47d0\x2da4ee\x2de1631327acac.scope/system.slice/mysql.service
+                     └─6 mysqld
+
+                     Jul 13 17:01:03 rkt-11efeefb-2e8d-47d0-a4ee-e1631327acac mysql[6]: 2016-07-13T17:01:03.047413Z 0 [Note]   - '::' resolves to '::';
+                     Jul 13 17:01:03 rkt-11efeefb-2e8d-47d0-a4ee-e1631327acac mysql[6]: 2016-07-13T17:01:03.047669Z 0 [Note] Server socket created on IP: '::'.
+                     Jul 13 17:01:03 rkt-11efeefb-2e8d-47d0-a4ee-e1631327acac mysql[6]: 2016-07-13T17:01:03.048279Z 0 [Note] InnoDB: Loading buffer pool(s) from /var/lib/mysql/ib_buffer_pool
+                     Jul 13 17:01:03 rkt-11efeefb-2e8d-47d0-a4ee-e1631327acac mysql[6]: 2016-07-13T17:01:03.080176Z 0 [Warning] 'db' entry 'sys mysql.sys@localhost' ignored in --skip-name-resolve mode.
+                     Jul 13 17:01:03 rkt-11efeefb-2e8d-47d0-a4ee-e1631327acac mysql[6]: 2016-07-13T17:01:03.085072Z 0 [Warning] 'proxies_priv' entry '@ root@localhost' ignored in --skip-name-resolve mode.
+                     Jul 13 17:01:03 rkt-11efeefb-2e8d-47d0-a4ee-e1631327acac mysql[6]: 2016-07-13T17:01:03.093485Z 0 [Note] InnoDB: Buffer pool(s) load completed at 160713 17:01:03
+                     Jul 13 17:01:03 rkt-11efeefb-2e8d-47d0-a4ee-e1631327acac mysql[6]: 2016-07-13T17:01:03.095603Z 0 [Warning] 'tables_priv' entry 'sys_config mysql.sys@localhost' ignored in --skip-name-resolve mode.
+                     Jul 13 17:01:03 rkt-11efeefb-2e8d-47d0-a4ee-e1631327acac mysql[6]: 2016-07-13T17:01:03.111731Z 0 [Note] Event Scheduler: Loaded 0 events
+                     Jul 13 17:01:03 rkt-11efeefb-2e8d-47d0-a4ee-e1631327acac mysql[6]: 2016-07-13T17:01:03.113195Z 0 [Note] mysqld: ready for connections.
+                     Jul 13 17:01:03 rkt-11efeefb-2e8d-47d0-a4ee-e1631327acac mysql[6]: Version: '5.7.13'  socket: '/var/run/mysqld/mysqld.sock'  port: 3306  MySQL Community Server (GPL)
+```
+
+## rkt app exec
+Executes a command inside an application.
+
+```bash
+rkt app exec <pod-uuid> --app=<app-name> <arguments> -- <command> <command-arguments>
+```
+
+# Entrypoints
+
+In order to facilitate the app-level operations API, 4 new entrypoints are introduced.
+These entrypoints are stage1 binaries that are capable of communicating with the stage1 supervisor.
+
+For example, the `app start` entrypoint in default stage1 flavor sends a `start` signal
+(similar to `systemctl start`) for the service files of the application to the systemd which is the stage1 supervisor.
+
+## rkt app inject
+
+`coreos.com/rkt/stage1/app/inject`
+
+1. prepare the application intended for injection, its rootfs and service files or else.
```

Comment:
The structure of this document is confusing; this section is ostensibly about entrypoints but this behaviour here is clearly pre-entrypoint. Can you just combine them for now, e.g. have one "app inject" section that describes what the stage0 and stage1 each do